src/main/java/org/parabot/core/ui/components/VerboseLoader.java

```diff
@@ -61,7 +61,7 @@ private VerboseLoader(String username, String password) {
         setOpaque(false);
 
         if (username != null && password != null) {
-            if (Core.inDebugMode() || manager.login(username, password, false)) {
+            if (Core.skipLogin() || manager.login(username, password, false)) {
```

Comment:
Have you tested opening the scripts page when in `skipLogin`?

Reply:
Yes, the scripts UI opens but of course is blank. If -loadlocal is on local scripts are displayed. It sends  an error message to `system.err`. I'll update the message to be more informative beyond what it currently is and also send a message to the `LoggerUI`. Probably worth sending a `UILog.alert()` too?